app/frontend/src/store/modules/auth.js

```diff
@@ -20,13 +23,20 @@ export default {
     // In most cases, when this becomes populated, we end up doing a redirect flow,
     // so when we return to the app, it is fresh again and undefined
     redirectUri: undefined,
+    showTokenExpiredWarningMSg: false,
```

Comment:
Can you change this to be `showTokenExpiredWarningMsg`?

Reply:
Okay